os/include/tinyara/mm/mm.h

```diff
@@ -249,6 +249,13 @@ typedef size_t mmsize_t;
 #error Unknown CONFIG_ARCH option, malloc debug feature wont work.
 #endif
 
+#ifdef CONFIG_DEBUG_MM_HEAPINFO
+/* Macro sets caller address of malloc API to mem chunk.
+ * If rapping malloc APT, have to called after calling the malloc API.
+ */
+#define DEBUG_SET_CALLER_ADDR(mem) heapinfo_set_caller_addr(mem, __builtin_return_address(0))
+#endif
```

Comment:
How about below?
```
#ifdef CONFIG_DEBUG_MM....
#define AA bb()
#else
#define AA
#endif
```
Then, user just calls the macro without the conditinal.

Reply:
Yes, I think so. Thanks, Fixed.